ASReviewLAB.qmd

```diff
@@ -105,7 +105,7 @@ bottom right corner.
 Select Validation Mode, fill out a project name and press ‘NEXT’. Note
 that you can fill out your name and a description as well.
 
-![](images/ASReviewLAB/Interface3.png){width=95%}
+![](https://raw.githubusercontent.com/asreview/asreview/773dab5946de0b4c61b8b07de1738f5c25a61e08/docs/images/setup_project_modes.png){width=95%}
```

Comment:
```suggestion
![](https://raw.githubusercontent.com/asreview/asreview/main/docs/images/setup_project_modes.png){width=95%}
```

Using main instead of commit hash might be the preferred solution.

Reply:
Makes sense, I'll also do it for the other links